bloom-housing/bloom
# feat: multiselect question endpoints

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/3440)


## Description
This creates all the endpoints for multiselect questions

## How Can This Be Tested/Reviewed?
Follow the readme instructions for how to stand the new backend up
You should be able to test things out using the openai site at http://localhost:3101/api

you should be able to:
create, get a list of, update, get an individual, and delete multiselect questions

permission is not implemented yet so don't worry about testing that yet

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
